cmd/acraserver/listener.go

```diff
@@ -107,30 +145,149 @@ func (server *SServer) handleConnection(connection net.Conn) {
 
 // start listening connections from proxy
 func (server *SServer) Start() {
-	listener, err := network.Listen(server.config.GetAcraConnectionString())
+	var connection net.Conn
+	var listener, err = network.Listen(server.config.GetAcraConnectionString())
 	if err != nil {
 		log.WithError(err).Errorln("can't start listen connections")
+		ErrorSignalChannel <- syscall.SIGTERM
```

Comment:
why not just return? listener is not created and the server hasn't any user connections.
And using global variable inside method looks a little bit strange. If we need it, better to get channel as function param or use struct field initialized somewhere else

Reply:
Passing signal channels as server's property